packages/Webkul/Shop/src/Http/Requests/CartAddressRequest.php

```diff
@@ -5,6 +5,7 @@
 use Illuminate\Foundation\Http\FormRequest;
 use Webkul\Core\Rules\PhoneNumber;
 use Webkul\Core\Rules\PostCode;
+use Webkul\Customer\Rules\VatIdRule;
```

Comment:
It would be helpful if you could upgrade this class to follow the latest Laravel conventions, as the current implementation is deprecated.

Reply:
If possible, please review the VAT Validator class as well. If you find any areas for improvement, feel free to make the necessary changes.